CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 ## Next release
 
+## 0.41.0 / 2020-07-29
+
+* [CHANGE] configmap-reload: Update to v0.4.0 (#3334)
+* [CHANGE] Update prometheus compatibility matrix to v2.19.2 (#3316)
+* [ENHANCEMENT] docs: update compatibility warning (#3369)
+* [ENHANCEMENT] Propagate Prometheus image version to statefulset (#3358)
+* [ENHANCEMENT] pkg: instrument resources being tracked by the operator (#3360)
+* [ENHANCEMENT] add SecretListWatchSelector to reduce memory and CPU footprint (#3355)
+* [ENHANCEMENT] Added support for configuring CA, cert, and key via secret or configmap. (#3249)
+* [ENHANCEMENT] go.mod: remove “replace” for client-go dep (#3332)
```

Comment:
This is not user-facing. In the past, we didn't include such changes.

Reply:
removed